src/mkdocstrings/handlers/python.py

```diff
@@ -45,6 +45,7 @@ class PythonRenderer(BaseRenderer):
         "show_source": True,
         "group_by_category": True,
         "heading_level": 2,
+        "sort_members": "lexical",
```

Comment:
I'd prefer the more general word "order" rather than "sort". I think "keeping original order" is no longer covered by the word "sort".

Also for "lexical", maybe you mean "lexicographic". But isn't "alphabetic" a much simpler term for this?

So maybe this would look something like:
```yaml
members_order: source
members_order: alphabetic
```
(but I'm not sure even about that specific naming, `members_order` looks a bit weird)

Reply:
Sphinx's autodoc calls it [`autodoc_member_order`](https://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html#confval-autodoc_member_order) so I guess `members_order` is fine. As for the values, I'd vote for `alphabetical` and `source` :slightly_smiling_face: